libs/model/src/aggregates/token/GetToken.query.ts

```diff
@@ -24,6 +24,19 @@ export function GetLaunchpadToken(): Query<typeof schemas.GetToken> {
         return null;
       }
 
+      const threadTokenSql = `
+        LEFT JOIN "Threads" AS TT ON TT.token_address = T.token_address
```

Comment:
Can narrow this to just `JOIN` since result will be empty if no thread exists anyways.